packages/mobile/src/web3/actions.ts

```diff
@@ -76,12 +76,12 @@ export const updateWeb3SyncProgress = (payload: {
 
 export const checkSyncProgress = () => ({ type: Actions.REQUEST_SYNC_PROGRESS })
 
-export function getLatestBlock() {
+export async function getLatestBlock() {
   Logger.debug(TAG, 'Getting latest block')
   return web3.eth.getBlock('latest')
 }
 
-export function getBlock(blockNumber: number) {
+export async function getBlock(blockNumber: number) {
```

Comment:
ditto

Reply:
Changed signature to `export function getBlock()`